### PR TITLE
GitHub: feat(meta): update security policy

### PR DESCRIPTION
**Pull request by JacobCoffee**
org-wide update of security policy

# PR Checklist

- [x] Have you followed the guidelines in [`CONTRIBUTING.rst`](https://github.com/starlite-api/starlite/blob/main/CONTRIBUTING.rst)?

### CODE REVIEW

**Review by provinzkraut — CHANGES_REQUESTED** [review]
I think we should encourage people to use [Security Advisories](https://github.com/starlite-api/starlite/security/advisories) instead of opening issues. That way potential vulnerabilities can be disclosed privately before we get a chance to fix them. It also comes with some other benefits to tackle and track those.

**Comment by JacobCoffee** [comment]
I am glad you mentioned this. It was in the original but I did not want to offend any previous decisions I wasn't aware of. The initial draft had something to the tune of "Please do not open an issue, please disclose to use privately" etc. Will get that fixed.

File: SECURITY.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+
+**Please do not report security vulnerabilities through public GitHub issues, discussions, or pull requests.**
+
+[Click to Open a Security Advisory Privately](https://github.com/starlite-api/starlite/security/advisories/new)
+
+If you believe you have found a security vulnerability in any Starlite-owned repository, please report it to us through coordinated disclosure:
+  * In the affected repository, browse to the **Security** tab, select **Advisories**, select "Report a vulnerability"
+      * ![image](https://user-images.githubusercontent.com/45884264/217041010-8fd6b96b-329d-4d8e-8838-9b5bf4e1a78d.png)
+  * Please Do **NOT** Creating an issue in the affected repository
+  * Please do **NOT** Private message or tag "@maintainer" on our [Discord server](https://discord.gg/MmcwxztmQb)
```
Comment (provinzkraut) [inline SECURITY.md]:
```suggestion
  * Please do **NOT** creating an issue in the affected repository
  * Please do **NOT** send a private message to or tag "@maintainer" on our [Discord server](https://discord.gg/MmcwxztmQb)
```

File: SECURITY.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
 # Security Policy
 
-## Reporting a Vulnerability
+Thanks for helping make Starlite safe for everyone.
 
-To report a vulnerability either create an issue on GitHub or tag "@maintainer" on our discord server
+## Security 
+
+Starlite takes the security of our projects and services seriously, including all of the open source code repositories managed through our GitHub organizations, such as [@starlite-api](https://github.com/starlite-api).
```
Comment (provinzkraut) [inline SECURITY.md:7]:
```suggestion
Starlite takes the security of our projects and services seriously, including all of the repositories managed by the [starlite-api organization](https://github.com/starlite-api).
```

File: SECURITY.md (hunk 1 above)
Comment (JacobCoffee) [reply SECURITY.md]: (meta) annoying github feature: you cant commit suggestions to the same file even if they dont affect the same line 
image


maybe that's what the batch feature is for? :(

will get this fixed

**Comment by JacobCoffee** [comment]
Before we approved, I recommend deleting entirely the `CODE_OF_CONDUCT.md` and `LICENSE` files from all repos.

Our organization-wide repo `.github` is special (https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile).

For example, if a repo existing under @starlite-api does not have a LICENSE, Code of Conduct, PR/Discussion/Issue template, etc. files - the meta `.github` repo takes over.

If there were some repo where we wanted a difference license, CoC, etc. we can add it to the repo. Repos with those files explicitly override the org-wide files.

But, deleting these from all repos allows us to house them in one repo, `.github`, and disseminate them throughout the org.

**Comment by Goldziher** [comment]
go ahead and delete whatever needs deletion in this PR.

**Comment by JacobCoffee** [comment]
Merging - deleted files in favor of:
- https://github.com/starlite-api/.github/security/policy
- https://github.com/starlite-api/.github/blob/main/CODE_OF_CONDUCT.md

**Pull request merged**